my-react-app/src/Apidata.js/Apidata.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from "react";
+
+function APidata(){
+    const[records,showRecords]=useState([]);
+    useEffect(()=>{
+        fetch("https://jsonplaceholder.typicode.com/users")
+        .then(response => response.json())
+        .then(data => showRecords(data))
+    },[])
+
+    return(
+        <div className="grid grid-cols-3">
+           
+               {records.map((list,id)=>(
+               <div className="flex m-5 ps-5 border-black" key={id}>
+
+                <div> Id:{list.id} 
+               <div>name :{list.name} </div>
+                <div>username:{list.username} </div>
```

Comment:
Use null check at every places where you are accessing the object keys (?. or &&)